main/realmain.cc

```diff
@@ -769,6 +766,14 @@ int realmain(int argc, char *argv[]) {
             // Second run: all the other files (the packageDB shouldn't change)
             pipeline::package(*gs, absl::Span<ast::ParsedFile>(nonPackageIndexed), opts, *workers);
 
+            {
+                // TODO(jez) Put this back after the call to `pipeline::package` in the stripePackages section
```

Comment:
Can we provide a little more color on _why_ we want this put back after that call?

Reply:
It doesn't do this yet, but in the future it will be the case that namer creates Package symbols in the symbol table. So by putting the call to `pipeline::name` over the subset of `__package.rb` files immediately after the call to `pipeline::package`, we will be able to populate the package DB before parsing and naming any other files.